airbyte-cdk/python/setup.py

```diff
@@ -54,6 +54,24 @@
         "requests",
         "vcrpy",
         "Deprecated~=1.2",
+        "Jinja2~=3.0",
```

Comment:
those dependencies are not required for egular cdk, can we move it under extras_require "docs" section and install it only when generating docs